tests/unit/utils/kubeapi/ingress.go

```diff
@@ -0,0 +1,42 @@
+package kubeapi
+
+import (
+	. "fmt"
+	. "github.com/onsi/gomega"
+	networkingV1 "k8s.io/api/networking/v1beta1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func FindIngressRuleByHttpPath(rules []networkingV1.IngressRule, path string) (*networkingV1.IngressRule, *networkingV1.HTTPIngressPath) {
```

Comment:
Can you add comments for this?

Reply:
Weird, I did. Guess they got lost in the cherry picking. I'll dig them up and restore.

Reply:
Here again, I think you were just looking at non-latest commits. The comments are present on the latest commit.